package.json

```diff
@@ -27,7 +27,8 @@
     "rimraf": "^2.6.2",
     "simple-git": "^1.80.1",
     "stylelint": "^8.2.0",
```

Comment:
bumped the `stylelint-config-standard` rule, as I noticed I was using a newer one.  This might be a bad idea - LMK if you'd rather I remove it